client-cli/Cargo.toml

```diff
@@ -26,4 +27,8 @@ cli-table = "0.2"
 zeroize = "1.1"
 base64 = "0.11"
 serde_json = "1.0"
+lazy_static = "1.4.0"
```

Comment:
People have started moving away from `lazy_static` crate. You can use [`once_cell`](https://crates.io/crates/once_cell) instead.